Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use the official Rust image as the base image
+FROM rust:1.81 AS builder
+
+# Install protobuf compiler
+RUN apt-get update && apt-get install -y protobuf-compiler
+
+# Create a new directory for the application
+WORKDIR /app
+
+# Copy the entire project
+COPY . .
```

Comment:
The issue mentions that is for deployments, so this dockerfile wouldn't necessary have the project it self. I think that we should pull from GitHub instead.